src/executables/csql.h

```diff
@@ -171,7 +173,8 @@ extern "C"
     CSQL_ERR_CANT_EDIT,
     CSQL_ERR_INFO_CMD_HELP,
     CSQL_ERR_CLASS_NAME_MISSED,
-    CSQL_ERR_FORMAT
+    CSQL_ERR_FORMAT,
+    CSQL_ERR_CONNECT
```

Comment:
error msg, **CSQL_ERR_CONNECT** is defined but not used.

Reply:
Ok. It will be removed thank you!